dev/CommandBarFlyout/CommandBarFlyoutCommandBar.cpp

```diff
@@ -765,10 +767,15 @@ void CommandBarFlyoutCommandBar::EnsureFocusedPrimaryCommand()
 
 void CommandBarFlyoutCommandBar::MoreButtonClicked(const winrt::IInspectable& sender, const winrt::RoutedEventArgs& args)
 {
-    // If the button get's invoked, we switch between secondary commands visible and secondary commands not visible
     m_secondaryCommandsExpectedToBeVisible = !m_secondaryCommandsExpectedToBeVisible;
 }
 
+void CommandBarFlyoutCommandBar::OverFlowPopupOpened(const winrt::IInspectable& sender, const winrt::IInspectable& args)
+{
+    // Overflow popup got opened, so we SecondaryCommands are expected to be visible
+    m_secondaryCommandsExpectedToBeVisible = true;
```

Comment:
Does this mean that we are dependent on the order that the OverFlowPopupOpened event and the moreButtonClicked events fire in?

Reply:
I am afraid, that this is the case, yes.